### PR TITLE
libxsmm support for CP2K

### PR DESCRIPTION
**Pull request by wpoely86**
@hajgato 

@boegel please review

### CODE REVIEW

File: easybuild/easyblocks/c/cp2k.py [hunk 1]
```diff
@@ -94,7 +95,7 @@ def extra_options():
             'extradflags': ['', "Extra DFLAGS to be added", CUSTOM],
             'ignore_regtest_fails': [False, ("Ignore failures in regression test "
                                              "(should be used with care)"), CUSTOM],
-            'maxtasks': [3, ("Maximum number of CP2K instances run at "
+            'maxtasks': [4, ("Maximum number of CP2K instances run at "
```
Comment (boegel) [inline easybuild/easyblocks/c/cp2k.py]: @wpoely86 why bump this, any specific reason?

**Comment by boegel** [comment]
Looks fine.

@wpoely86: points to easyconfigs that need this?

File: easybuild/easyblocks/c/cp2k.py (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyblocks/c/cp2k.py]: this is for the hybrid build. It must be a multiple of two: each process gets two threads.

Comment (boegel) [inline easybuild/easyblocks/c/cp2k.py]: OK, fine then

**Comment by hajgato** [comment]
@boegel ^

**Comment by wpoely86** [comment]
Also needed for hpcugent/easybuild-easyconfigs#2889

**Comment by boegel** [comment]
Thanks for the pointers... I'd like to see a successful test report for either of those (#3099 first?) before we merge this in.

**Comment by boegel** [comment]
tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/3099, good to go, thanks @hajgato and @wpoely86!

**Pull request merged**